website/static/js/commentpane.js

```diff
@@ -11,7 +11,10 @@
         onOpen: function() {}
     };
 
-    var CommentPane = CommentPane || function(selector, options) {
+    /*
+       mode: 'widget', 'pane' or 'page'
+     */
+    var CommentPane = CommentPane || function(selector, host_page, host_name, mode, options) {
```

Comment:
@sloria <code>mode</code> includes "page" (<code>/<pid>/discussion/</code>), "pane" (the comment pane you are familiar with) and "widget" (only 5 recent discussions show up.) The <code>host_page</code> attribute is more referred to <code>page</code> in the js file. It is useful in <code>/<pid>/discussion/</code> too:
![screen shot 2015-03-30 at 3 47 27 pm](https://cloud.githubusercontent.com/assets/8628591/6905177/335e05c6-d6f4-11e4-96c0-918758a68d7a.png)